eclipse-theia/theia
# fix: fix eslint-plugin runtime import check rule

#### What it does
fix eslint-plugin runtime import check rule

#### How to test

run: yarn lint

case:
in a common folder like: src/common/**.ts ，the rule only matched  /browser/, unmatched /browser;：
```
import { FrontendApplicationContribution } from '@theia/core/lib/browser'; 
```


 [if (matchedImportRule.restricted.some(restricted => module.includes(`/${restricted}/`))) {](https://github.com/eclipse-theia/theia/blob/f81ef73a5379c938f57aad5ba0a109964169b4a4/dev-packages/eslint-plugin/rules/runtime-import-check.js#L111)

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
